README.md

```diff
@@ -1,2 +1,9 @@
-# lib-python-shapely
-Manipulation and analysis of geometric objects
+Shapely Python extension for Unikraft
+=====================================
+
+This is a port of the [Shapely](https://pypi.org/project/shapely/) Python
+extension to Unikraft.
+
+Please refer to the `README.md` as well as the documentation in the
+`doc/` subdirectory of the main unikraft repository for further
+information.
```

Comment:
Change this to md format instead of rst.
Use one line per sentence and one sentence per line.